Update test262 test cases

Gumichocopengin8 opened the pull request:
#### 25ad4324c5f6bab67bc1b5c9567258ba4462262a
<pre>
Update test262 test cases
<a href="https://bugs.webkit.org/show_bug.cgi?id=289874">https://bugs.webkit.org/show_bug.cgi?id=289874</a>

Reviewed by Sosuke Suzuki.

Update test262 test cases to align with <a href="https://github.com/tc39/test262/commit/4ba104a21c5f8ede20ee5105bc89fa9693c2bfe1">https://github.com/tc39/test262/commit/4ba104a21c5f8ede20ee5105bc89fa9693c2bfe1</a>

* JSTests/test262/expectations.yaml:
* JSTests/test262/harness/hidden-constructors.js: Removed.
* JSTests/test262/harness/temporalHelpers.js:
(TemporalHelpers.ISO.plainMonthDayStringsInvalid):
(TemporalHelpers.ISO.plainMonthDayStringsValid):
* JSTests/test262/harness/wellKnownIntrinsicObjects.js:
(getWellKnownIntrinsicObject):
* JSTests/test262/latest-changes-summary.txt:
* JSTests/test262/test/built-ins/AsyncFunction/is-a-constructor.js:
* JSTests/test262/test/built-ins/AsyncGeneratorFunction/is-a-constructor.js:
* JSTests/test262/test/built-ins/AsyncIteratorPrototype/Symbol.asyncDispose/invokes-return.js: Added.
(asyncTest.async generator):
(asyncTest.async returnCalled):
(asyncTest.iter.return):
(asyncTest):
* JSTests/test262/test/built-ins/AsyncIteratorPrototype/Symbol.asyncDispose/return-val.js: Added.
(async generator):
* JSTests/test262/test/built-ins/Date/prototype/setUTCHours/this-value-valid-date-hour.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCHours/this-value-valid-date-min.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCHours/this-value-valid-date-ms.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCHours/this-value-valid-date-sec.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCMilliseconds/this-value-valid-date.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCMinutes/this-value-valid-date.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCMonth/this-value-valid-date-date.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCMonth/this-value-valid-date-month.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCSeconds/this-value-valid-date-ms.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCSeconds/this-value-valid-date-sec.js: Added.
* JSTests/test262/test/built-ins/GeneratorFunction/is-a-constructor.js:
* JSTests/test262/test/built-ins/Iterator/prototype/Symbol.dispose/invokes-return.js: Added.
(iter.return):
* JSTests/test262/test/built-ins/Iterator/prototype/Symbol.dispose/return-val.js: Added.
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/message-method-prop-cast.js: Added.
(toString):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/message-method-prop.js: Added.
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/message-tostring-abrupt-symbol.js: Added.
(case2.Symbol.toPrimitive):
(case2.toString):
(case2.valueOf):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/message-tostring-abrupt.js: Added.
(case1.Symbol.toPrimitive):
(case1.toString):
(case1.valueOf):
(case2.toString):
(case2.valueOf):
(case3.valueOf):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/newtarget-is-undefined.js: Added.
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/newtarget-proto-custom.js: Added.
(newt.new.Proxy):
(newt.new.Proxy.get if):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/newtarget-proto-fallback.js: Added.
(const.value.of.values.get t):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/newtarget-proto.js: Added.
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/order-of-args-evaluation.js: Added.
(const.message.toString):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/prototype/errors-absent-on-prototype.js: Added.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-negative-cases.js: Renamed from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-positive-cases.js: Renamed from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-plus-quantifier-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-negative-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-plus-quantifier-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-plus-quantifier.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-positive-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-negative-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-plus-quantifier-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-plus-quantifier.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-positive-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-negative-cases.js: Renamed from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-plus-quantifier.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-plus-quantifier-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-plus-quantifier.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-positive-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape-negative-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape-plus-quantifier-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape-plus-quantifier.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape-positive-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape-negative-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape-plus-quantifier-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape-plus-quantifier.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape-positive-cases.js: Renamed from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape.js: Removed.
* JSTests/test262/test/built-ins/String/fromCodePoint/argument-is-Symbol.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/argument-is-not-integer.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/argument-not-coercible.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/arguments-is-empty.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/number-is-out-of-range.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/return-string-value.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/to-number-conversions.js:
* JSTests/test262/test/built-ins/String/prototype/match/cstm-matcher-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/match/cstm-matcher-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/match/cstm-matcher-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/match/cstm-matcher-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/matchAll/cstm-matchall-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/matchAll/cstm-matchall-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/matchAll/cstm-matchall-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/matchAll/cstm-matchall-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replace/cstm-replace-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replace/cstm-replace-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replace/cstm-replace-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replace/cstm-replace-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replaceAll/cstm-replaceall-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replaceAll/cstm-replaceall-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replaceAll/cstm-replaceall-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replaceAll/cstm-replaceall-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/search/cstm-search-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/search/cstm-search-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/search/cstm-search-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/search/cstm-search-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/split/cstm-split-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/split/cstm-split-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/split/cstm-split-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/split/cstm-split-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/Temporal/Duration/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/Duration/prototype/total/precision-exact-mathematical-values-5.js:
(assert.sameValue):
* JSTests/test262/test/built-ins/Temporal/Duration/prototype/total/relativeto-propertybag-invalid-offset-string.js:
(badOffsets.forEach):
* JSTests/test262/test/built-ins/Temporal/Instant/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/PlainDate/prototype/era/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDate/prototype/era/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainDate/prototype/era/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDate/prototype/era/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainDate/prototype/eraYear/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDate/prototype/eraYear/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainDate/prototype/eraYear/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDate/prototype/eraYear/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainDate/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/PlainDateTime/prototype/era/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDateTime/prototype/era/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainDateTime/prototype/era/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDateTime/prototype/era/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainDateTime/prototype/eraYear/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDateTime/prototype/eraYear/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainDateTime/prototype/eraYear/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDateTime/prototype/eraYear/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainDateTime/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/PlainMonthDay/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/PlainTime/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/PlainYearMonth/prototype/era/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainYearMonth/prototype/era/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainYearMonth/prototype/era/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainYearMonth/prototype/era/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainYearMonth/prototype/eraYear/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainYearMonth/prototype/eraYear/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainYearMonth/prototype/eraYear/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainYearMonth/prototype/eraYear/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainYearMonth/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/ZonedDateTime/prototype/era/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/ZonedDateTime/prototype/era/branding.js.
* JSTests/test262/test/built-ins/Temporal/ZonedDateTime/prototype/era/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/ZonedDateTime/prototype/era/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/ZonedDateTime/prototype/eraYear/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/ZonedDateTime/prototype/eraYear/branding.js.
* JSTests/test262/test/built-ins/Temporal/ZonedDateTime/prototype/eraYear/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/ZonedDateTime/prototype/eraYear/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/ZonedDateTime/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/decodeURIComponent/throw-URIError.js: Added.
(CHECK1):
(CHECK2):
(CHECK3):
(CHECK4):
(CHECK5):
* JSTests/test262/test/harness/wellKnownIntrinsicObjects.js: Added.
* JSTests/test262/test/intl402/DateTimeFormat/canonicalize-utc-timezone.js:
* JSTests/test262/test/intl402/DateTimeFormat/prototype/formatRangeToParts/to-datetime-formattable-with-different-arg-kinds.js:
* JSTests/test262/test/intl402/DurationFormat/prototype/format/style-digital-largenumber-en.js: Added.
(format.in.digit.set useGrouping):
* JSTests/test262/test/intl402/DurationFormat/prototype/format/taint-temporal-duration-prototype.js: Added.
* JSTests/test262/test/intl402/DurationFormat/prototype/format/temporal-duration-object-arg.js: Added.
* JSTests/test262/test/intl402/DurationFormat/prototype/format/temporal-duration-string-arg.js: Added.
* JSTests/test262/test/intl402/DurationFormat/prototype/formatToParts/taint-temporal-duration-prototype.js: Added.
(assertSameParts):
* JSTests/test262/test/intl402/DurationFormat/prototype/formatToParts/temporal-duration-object-arg.js: Added.
(assertSameParts):
* JSTests/test262/test/intl402/DurationFormat/prototype/formatToParts/temporal-duration-string-arg.js: Added.
(assertSameParts):
* JSTests/test262/test/intl402/Locale/getters.js:
* JSTests/test262/test/intl402/NumberFormat/currency-digits-nonstandard-notation.js:
* JSTests/test262/test/intl402/Temporal/Duration/prototype/toLocaleString/returns-same-results-as-DurationFormat.js: Added.
* JSTests/test262/test/intl402/Temporal/Duration/prototype/toLocaleString/taint-duration-prototype.js: Added.
* JSTests/test262/test/intl402/Temporal/ZonedDateTime/prototype/equals/canonicalize-utc-timezone.js: Added.
* JSTests/test262/test/language/expressions/async-arrow-function/prototype.js:
* JSTests/test262/test/language/import/import-defer/errors/resolution-error/import-defer-of-missing-module-fails.js:
(asyncTest.async globalThis):
(asyncTest):
* JSTests/test262/test/language/import/import-defer/errors/resolution-error/main_FIXTURE.js: Added.
* JSTests/test262/test/language/module-code/import-attributes/allow-nlt-before-with.js:
* JSTests/test262/test/language/module-code/import-attributes/ensure-linking-error_FIXTURE.js: Copied from JSTests/test262/test/language/module-code/resources/ensure-linking-error_FIXTURE.js.
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-key-identifiername.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-key-string-double.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-key-string-single.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-many.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-newlines.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-trlng-comma.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-value-string-double.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-value-string-single.js:
* JSTests/test262/test/language/module-code/source-phase-import/ensure-linking-error_FIXTURE.js: Renamed from JSTests/test262/test/language/module-code/resources/ensure-linking-error_FIXTURE.js.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-binding-name-2.js: Removed.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-binding-name-2_FIXTURE.js: Added.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-binding-name.js: Removed.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-binding-name_FIXTURE.js: Added.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-newlines.js: Removed.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-newlines_FIXTURE.js: Added.
* JSTests/test262/test/language/module-code/source-phase-import/import-source.js: Added.
(assertImportSourceResolutionFailure):
* JSTests/test262/test/staging/Intl402/Temporal/old/addition-across-lunisolar-leap-months-chinese.js: Added.
* JSTests/test262/test/staging/Intl402/Temporal/old/addition-across-lunisolar-leap-months-hebrew.js: Added.
* JSTests/test262/test/staging/Intl402/Temporal/old/addition-across-lunisolar-leap-months.js: Removed.
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-buddhist.js: Added.
(compareFormatToPartsSnapshot):
(const.yearOneDay):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-chinese.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
(of.daysInMonthCases.i.monthCode.endsWith):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-coptic.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-dangi.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
(of.daysInMonthCases.i.monthCode.endsWith):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-ethioaa.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-ethiopic.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-gregory.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-hebrew.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
(of.daysInMonthCases.i.monthCode.endsWith):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-indian.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamic-civil.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamic-rgsa.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamic-tbla.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamic-umalqura.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamic.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamicc.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-iso8601.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-japanese.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-persian.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-roc.js: Added.
(compareFormatToPartsSnapshot):
(const.yearOneDay):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars.js: Removed.
* JSTests/test262/test/staging/explicit-resource-management/async-disposal-from-sync-method-returning-a-promise.js: Added.
(asyncTest.async TestAsyncDisposalWithSyncMethodReturningAPromise):
* JSTests/test262/test/staging/explicit-resource-management/await-using-in-top-level-module.js: Added.
(await.using.x.Symbol.asyncDispose):
* JSTests/test262/test/staging/explicit-resource-management/disposable-stack-re-entry.js: Added.
* JSTests/test262/test/staging/explicit-resource-management/using-with-null-or-undefined.js:
* JSTests/test262/test/staging/sm/Array/change-array-by-copy-errors-from-correct-realm.js:
(test):
* JSTests/test262/test/staging/sm/Iterator/prototype/every/check-fn-after-getting-iterator.js:
* JSTests/test262/test/staging/sm/Iterator/prototype/find/check-fn-after-getting-iterator.js:
* JSTests/test262/test/staging/sm/Iterator/prototype/forEach/check-fn-after-getting-iterator.js:
* JSTests/test262/test/staging/sm/Iterator/prototype/reduce/check-fn-after-getting-iterator.js:
* JSTests/test262/test/staging/sm/Iterator/prototype/some/check-fn-after-getting-iterator.js:
* JSTests/test262/test/staging/sm/extensions/inc-dec-functioncall.js: Removed.
* JSTests/test262/test/staging/sm/regress/regress-609617.js: Removed.
* JSTests/test262/test/staging/sm/statements/for-of-iterator-close.js:
* JSTests/test262/test262-Revision.txt:

Canonical link: <a href="https://commits.webkit.org/292875@main">https://commits.webkit.org/292875@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/00a8f7522c69032471a5473408b7d53c615ed435

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/97093 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/16716 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/6924 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/102174 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/47618 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/17008 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/25166 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/73962 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/31176 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/100096 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/12834 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/87812 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/54304 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/12588 "Passed tests") | | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/46948 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/89766 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/82651 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/5752 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/104196 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/95714 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/24167 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/17629 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/83025 "Passed tests") | 
| | [❌ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/24544 "Hash 00a8f752 for PR 42550 does not build (failure)") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/83926 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/82426 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/20792 "Built successfully and passed tests") | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-2-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/4647 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/17672 "The change is no longer eligible for processing.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/24132 "Built successfully") | | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/119341 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/23955 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/33521 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/27267 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/25528 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

sosukesuzuki reviewed (changes requested):
I think there should only be one commit in the PR, just like the others. I always use `git webkit commit --amend` to amend my commits.

Did you run `test262-import`? The file `test262-Revition.txt` should be updated, but I don’t see it in your changes.

Did you run `test262-runner` with the `--save` option? `expectations.yaml` should be updated as well (it sometimes isn’t, but it usually is). Also, if your local time zone is not PST, you should run it with the `TZ=PST` environment variable.

Finally, I recommend running it on the latest version of macOS.

sosukesuzuki reviewed (approved):
r=me

Gumichocopengin8 committed:
Update test262 test cases
https://bugs.webkit.org/show_bug.cgi?id=289874

Reviewed by Sosuke Suzuki.

Update test262 test cases to align with https://github.com/tc39/test262/commit/4ba104a21c5f8ede20ee5105bc89fa9693c2bfe1

* JSTests/test262/expectations.yaml:
* JSTests/test262/harness/hidden-constructors.js: Removed.
* JSTests/test262/harness/temporalHelpers.js:
(TemporalHelpers.ISO.plainMonthDayStringsInvalid):
(TemporalHelpers.ISO.plainMonthDayStringsValid):
* JSTests/test262/harness/wellKnownIntrinsicObjects.js:
(getWellKnownIntrinsicObject):
* JSTests/test262/latest-changes-summary.txt:
* JSTests/test262/test/built-ins/AsyncFunction/is-a-constructor.js:
* JSTests/test262/test/built-ins/AsyncGeneratorFunction/is-a-constructor.js:
* JSTests/test262/test/built-ins/AsyncIteratorPrototype/Symbol.asyncDispose/invokes-return.js: Added.
(asyncTest.async generator):
(asyncTest.async returnCalled):
(asyncTest.iter.return):
(asyncTest):
* JSTests/test262/test/built-ins/AsyncIteratorPrototype/Symbol.asyncDispose/return-val.js: Added.
(async generator):
* JSTests/test262/test/built-ins/Date/prototype/setUTCHours/this-value-valid-date-hour.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCHours/this-value-valid-date-min.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCHours/this-value-valid-date-ms.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCHours/this-value-valid-date-sec.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCMilliseconds/this-value-valid-date.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCMinutes/this-value-valid-date.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCMonth/this-value-valid-date-date.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCMonth/this-value-valid-date-month.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCSeconds/this-value-valid-date-ms.js: Added.
* JSTests/test262/test/built-ins/Date/prototype/setUTCSeconds/this-value-valid-date-sec.js: Added.
* JSTests/test262/test/built-ins/GeneratorFunction/is-a-constructor.js:
* JSTests/test262/test/built-ins/Iterator/prototype/Symbol.dispose/invokes-return.js: Added.
(iter.return):
* JSTests/test262/test/built-ins/Iterator/prototype/Symbol.dispose/return-val.js: Added.
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/message-method-prop-cast.js: Added.
(toString):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/message-method-prop.js: Added.
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/message-tostring-abrupt-symbol.js: Added.
(case2.Symbol.toPrimitive):
(case2.toString):
(case2.valueOf):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/message-tostring-abrupt.js: Added.
(case1.Symbol.toPrimitive):
(case1.toString):
(case1.valueOf):
(case2.toString):
(case2.valueOf):
(case3.valueOf):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/newtarget-is-undefined.js: Added.
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/newtarget-proto-custom.js: Added.
(newt.new.Proxy):
(newt.new.Proxy.get if):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/newtarget-proto-fallback.js: Added.
(const.value.of.values.get t):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/newtarget-proto.js: Added.
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/order-of-args-evaluation.js: Added.
(const.message.toString):
* JSTests/test262/test/built-ins/NativeErrors/SuppressedError/prototype/errors-absent-on-prototype.js: Added.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-negative-cases.js: Renamed from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-positive-cases.js: Renamed from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-plus-quantifier-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-negative-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-plus-quantifier-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-plus-quantifier.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-positive-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-negative-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-plus-quantifier-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-plus-quantifier.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-positive-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-negative-cases.js: Renamed from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-plus-quantifier.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-plus-quantifier-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-plus-quantifier.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-positive-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape-negative-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape-plus-quantifier-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape-plus-quantifier.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape-positive-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-whitespace-class-escape.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape-negative-cases.js: Copied from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape-plus-quantifier-flags-u.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape-plus-quantifier.js: Removed.
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape-positive-cases.js: Renamed from JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-flags-u.js.
(const.regex.of.regexes.regex.test):
* JSTests/test262/test/built-ins/RegExp/CharacterClassEscapes/character-class-word-class-escape.js: Removed.
* JSTests/test262/test/built-ins/String/fromCodePoint/argument-is-Symbol.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/argument-is-not-integer.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/argument-not-coercible.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/arguments-is-empty.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/number-is-out-of-range.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/return-string-value.js:
* JSTests/test262/test/built-ins/String/fromCodePoint/to-number-conversions.js:
* JSTests/test262/test/built-ins/String/prototype/match/cstm-matcher-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/match/cstm-matcher-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/match/cstm-matcher-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/match/cstm-matcher-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/matchAll/cstm-matchall-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/matchAll/cstm-matchall-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/matchAll/cstm-matchall-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/matchAll/cstm-matchall-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replace/cstm-replace-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replace/cstm-replace-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replace/cstm-replace-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replace/cstm-replace-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replaceAll/cstm-replaceall-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replaceAll/cstm-replaceall-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replaceAll/cstm-replaceall-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/replaceAll/cstm-replaceall-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/search/cstm-search-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/search/cstm-search-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/search/cstm-search-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/search/cstm-search-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/split/cstm-split-on-bigint-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/split/cstm-split-on-boolean-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/split/cstm-split-on-number-primitive.js: Added.
* JSTests/test262/test/built-ins/String/prototype/split/cstm-split-on-string-primitive.js: Added.
* JSTests/test262/test/built-ins/Temporal/Duration/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/Duration/prototype/total/precision-exact-mathematical-values-5.js:
(assert.sameValue):
* JSTests/test262/test/built-ins/Temporal/Duration/prototype/total/relativeto-propertybag-invalid-offset-string.js:
(badOffsets.forEach):
* JSTests/test262/test/built-ins/Temporal/Instant/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/PlainDate/prototype/era/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDate/prototype/era/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainDate/prototype/era/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDate/prototype/era/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainDate/prototype/eraYear/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDate/prototype/eraYear/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainDate/prototype/eraYear/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDate/prototype/eraYear/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainDate/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/PlainDateTime/prototype/era/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDateTime/prototype/era/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainDateTime/prototype/era/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDateTime/prototype/era/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainDateTime/prototype/eraYear/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDateTime/prototype/eraYear/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainDateTime/prototype/eraYear/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainDateTime/prototype/eraYear/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainDateTime/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/PlainMonthDay/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/PlainTime/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/PlainYearMonth/prototype/era/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainYearMonth/prototype/era/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainYearMonth/prototype/era/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainYearMonth/prototype/era/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainYearMonth/prototype/eraYear/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainYearMonth/prototype/eraYear/branding.js.
* JSTests/test262/test/built-ins/Temporal/PlainYearMonth/prototype/eraYear/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/PlainYearMonth/prototype/eraYear/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/PlainYearMonth/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/Temporal/ZonedDateTime/prototype/era/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/ZonedDateTime/prototype/era/branding.js.
* JSTests/test262/test/built-ins/Temporal/ZonedDateTime/prototype/era/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/ZonedDateTime/prototype/era/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/ZonedDateTime/prototype/eraYear/branding.js: Renamed from JSTests/test262/test/intl402/Temporal/ZonedDateTime/prototype/eraYear/branding.js.
* JSTests/test262/test/built-ins/Temporal/ZonedDateTime/prototype/eraYear/prop-desc.js: Renamed from JSTests/test262/test/intl402/Temporal/ZonedDateTime/prototype/eraYear/prop-desc.js.
* JSTests/test262/test/built-ins/Temporal/ZonedDateTime/prototype/prop-desc.js:
* JSTests/test262/test/built-ins/decodeURIComponent/throw-URIError.js: Added.
(CHECK1):
(CHECK2):
(CHECK3):
(CHECK4):
(CHECK5):
* JSTests/test262/test/harness/wellKnownIntrinsicObjects.js: Added.
* JSTests/test262/test/intl402/DateTimeFormat/canonicalize-utc-timezone.js:
* JSTests/test262/test/intl402/DateTimeFormat/prototype/formatRangeToParts/to-datetime-formattable-with-different-arg-kinds.js:
* JSTests/test262/test/intl402/DurationFormat/prototype/format/style-digital-largenumber-en.js: Added.
(format.in.digit.set useGrouping):
* JSTests/test262/test/intl402/DurationFormat/prototype/format/taint-temporal-duration-prototype.js: Added.
* JSTests/test262/test/intl402/DurationFormat/prototype/format/temporal-duration-object-arg.js: Added.
* JSTests/test262/test/intl402/DurationFormat/prototype/format/temporal-duration-string-arg.js: Added.
* JSTests/test262/test/intl402/DurationFormat/prototype/formatToParts/taint-temporal-duration-prototype.js: Added.
(assertSameParts):
* JSTests/test262/test/intl402/DurationFormat/prototype/formatToParts/temporal-duration-object-arg.js: Added.
(assertSameParts):
* JSTests/test262/test/intl402/DurationFormat/prototype/formatToParts/temporal-duration-string-arg.js: Added.
(assertSameParts):
* JSTests/test262/test/intl402/Locale/getters.js:
* JSTests/test262/test/intl402/NumberFormat/currency-digits-nonstandard-notation.js:
* JSTests/test262/test/intl402/Temporal/Duration/prototype/toLocaleString/returns-same-results-as-DurationFormat.js: Added.
* JSTests/test262/test/intl402/Temporal/Duration/prototype/toLocaleString/taint-duration-prototype.js: Added.
* JSTests/test262/test/intl402/Temporal/ZonedDateTime/prototype/equals/canonicalize-utc-timezone.js: Added.
* JSTests/test262/test/language/expressions/async-arrow-function/prototype.js:
* JSTests/test262/test/language/import/import-defer/errors/resolution-error/import-defer-of-missing-module-fails.js:
(asyncTest.async globalThis):
(asyncTest):
* JSTests/test262/test/language/import/import-defer/errors/resolution-error/main_FIXTURE.js: Added.
* JSTests/test262/test/language/module-code/import-attributes/allow-nlt-before-with.js:
* JSTests/test262/test/language/module-code/import-attributes/ensure-linking-error_FIXTURE.js: Copied from JSTests/test262/test/language/module-code/resources/ensure-linking-error_FIXTURE.js.
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-key-identifiername.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-key-string-double.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-key-string-single.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-many.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-newlines.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-trlng-comma.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-value-string-double.js:
* JSTests/test262/test/language/module-code/import-attributes/import-attribute-value-string-single.js:
* JSTests/test262/test/language/module-code/source-phase-import/ensure-linking-error_FIXTURE.js: Renamed from JSTests/test262/test/language/module-code/resources/ensure-linking-error_FIXTURE.js.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-binding-name-2.js: Removed.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-binding-name-2_FIXTURE.js: Added.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-binding-name.js: Removed.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-binding-name_FIXTURE.js: Added.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-newlines.js: Removed.
* JSTests/test262/test/language/module-code/source-phase-import/import-source-newlines_FIXTURE.js: Added.
* JSTests/test262/test/language/module-code/source-phase-import/import-source.js: Added.
(assertImportSourceResolutionFailure):
* JSTests/test262/test/staging/Intl402/Temporal/old/addition-across-lunisolar-leap-months-chinese.js: Added.
* JSTests/test262/test/staging/Intl402/Temporal/old/addition-across-lunisolar-leap-months-hebrew.js: Added.
* JSTests/test262/test/staging/Intl402/Temporal/old/addition-across-lunisolar-leap-months.js: Removed.
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-buddhist.js: Added.
(compareFormatToPartsSnapshot):
(const.yearOneDay):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-chinese.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
(of.daysInMonthCases.i.monthCode.endsWith):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-coptic.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-dangi.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
(of.daysInMonthCases.i.monthCode.endsWith):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-ethioaa.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-ethiopic.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-gregory.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-hebrew.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
(of.daysInMonthCases.i.monthCode.endsWith):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-indian.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamic-civil.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamic-rgsa.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamic-tbla.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamic-umalqura.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamic.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-islamicc.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-iso8601.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-japanese.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-persian.js: Added.
(compareFormatToPartsSnapshot):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars-roc.js: Added.
(compareFormatToPartsSnapshot):
(const.yearOneDay):
(result.of.Object.entries):
(of.Object.entries):
* JSTests/test262/test/staging/Intl402/Temporal/old/non-iso-calendars.js: Removed.
* JSTests/test262/test/staging/explicit-resource-management/async-disposal-from-sync-method-returning-a-promise.js: Added.
(asyncTest.async TestAsyncDisposalWithSyncMethodReturningAPromise):
* JSTests/test262/test/staging/explicit-resource-management/await-using-in-top-level-module.js: Added.
(await.using.x.Symbol.asyncDispose):
* JSTests/test262/test/staging/explicit-resource-management/disposable-stack-re-entry.js: Added.
* JSTests/test262/test/staging/explicit-resource-management/using-with-null-or-undefined.js:
* JSTests/test262/test/staging/sm/Array/change-array-by-copy-errors-from-correct-realm.js:
(test):
* JSTests/test262/test/staging/sm/Iterator/prototype/every/check-fn-after-getting-iterator.js:
* JSTests/test262/test/staging/sm/Iterator/prototype/find/check-fn-after-getting-iterator.js:
* JSTests/test262/test/staging/sm/Iterator/prototype/forEach/check-fn-after-getting-iterator.js:
* JSTests/test262/test/staging/sm/Iterator/prototype/reduce/check-fn-after-getting-iterator.js:
* JSTests/test262/test/staging/sm/Iterator/prototype/some/check-fn-after-getting-iterator.js:
* JSTests/test262/test/staging/sm/extensions/inc-dec-functioncall.js: Removed.
* JSTests/test262/test/staging/sm/regress/regress-609617.js: Removed.
* JSTests/test262/test/staging/sm/statements/for-of-iterator-close.js:
* JSTests/test262/test262-Revision.txt:

Canonical link: https://commits.webkit.org/292875@main
